### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SiansandaTamara**
### The following were implemented###
-   Created routes and view components (Rockets, Missions, My Profile).

-  Used NavLink for the page navigation links and styled active class to indicate which section/page user is currently on (underline active navigation link).
 
-  Created directories for all Redux state slice files (rockets, missions, ).

-   Fetched data and updated Redux store

-  Upon first render fetched data from the SpaceX API endpoints:

-  Once the data is fetched,  they dispatche an action to store the selected data in Redux store:

-  Used useSelector() Redux Hook to select the state slices .
-  Added testing libaries

### CODE REVIEW

File: src/Components/Rockects.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      ))}
+    </div>
+  );
+};
+Rockets.propTypes = {
+  reserveRocketClick: PropTypes.func.isRequired,
+  cancelReserveClick: PropTypes.func.isRequired,
+};
+
+export default Rockets;
```
Comment (DammyShittu) [inline src/Components/Rockects.js:37]: - [ ] This file seems to be a duplicate of `src/Components/Rockets.js` as they have exactly the code and functionality. It is also not imported into any file as well. Kindly delete the as it is unnecessary

File: src/redux/rockets/rockets.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+const initialState = [];
+
+export const displayRocket = () => async (dispatch) => {
+  const response = await axios.get(baseUrl);
+  dispatch({
+    type: DISPLAY_ROCKET,
+    payload: response.data,
+  });
+};
```
Comment (DammyShittu) [inline src/redux/rockets/rockets.js:16]: - [ ] Just as you have done with the missions data when your rockets data is fetched you are expected to filter your data and avoid saving all the data in the state.

  According to the guidelines, you are required to only save `id`, `rocket_name`, `description`, and `flickr_images`. Kindly do that.

  Missions state (correct) | Rocket state (incorrect)
  :---:|:---:
  pere-missions | pere-filter

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi team,

✔️ Proper documentation
✔️ App styling

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/rockets/rockets.js (hunk 2 above)
Comment (Stanleeeeee) [reply src/redux/rockets/rockets.js:16]: @DammyShittu Please we don't understand what you mean here. Can you kindly throw more light

Comment (zhadier) [reply src/redux/rockets/rockets.js:16]: - [ ] What the previous reviewer meant for you to do here is to only save the data you will be using use inside the Redux store. Right now, in the rockets reducer you are storing the entire data you are getting from the API. However, the data you are using is only `id`, `rocket_name`, `description` and `flickr_images` so it makes sense to only store this data from the API inside the store. Remember that the larger the Array, the more time your app will take to find the required data and the slower your application will be. Kindly make these changes, and if you still need further guidance I'd be more than happy to help 🌦️

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @SiansandaTamara & @Stanleeeeee   👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🌤️ 



 ## Highlights

✔️ No linter errors
✔️ GitFlow was setup correctly
✔️ Professional pull request title and description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/rockets/rockets.js (hunk 2 above)
Comment (Stanleeeeee) [reply src/redux/rockets/rockets.js:16]: @zhadier, can we connect with you on slack please.

Comment (zhadier) [reply src/redux/rockets/rockets.js:16]: Of course, my slack handle is Zeeshan Haider.

**Review by andersonlebon — APPROVED** [review]
## Hi @SiansandaTamara 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**